### ISSUE TITLE
Can you add the block requester feature to the Queue page?

### ISSUE THREAD

**Issue by awoegno** [Anveio/mturk-engine#16]
Thank you so much for this script! It is WAY better than fiddling around with that clunky user UNfriendly Mturk Suite. What a lifesaver!

Is there a way to add the "block" feature to the Queue area? When I come across a bad HIT that I am in the middle of, I can return it from the Queue page, but I also want to block the requester from there too. Currently, you have to go back to the Search page and block the requester there. But the HIT is usually erased from there since I usually have the search function going, so I can't block them from there either.

Or if there was a way to manually add a person to the block list on the block page, that would work too if it is easier to do that. That way I can add also the people I blocked on HITScraper to Mturk Engine.

Other than that, this is great!

**Comment by Anveio** [comment]
Hi @awoegno, thanks for submitting this feature request. I like this idea. I've been thinking of ways to add features to the queue tab and this makes a lot of sense.

Regarding your first point, I'm glad I was able to make a script that works better for you. I happen to like Mturk Suite, the breadth of features it has is impressive. I do make an effort to be as user friendly as possible, though, thanks for noticing. 😃

**Comment by Anveio** [comment]
This feature landed in v1.7.0. Feel free to open a separate issue if you run into a bug with it.
